lingua_franca/parse.py

```diff
@@ -158,7 +159,33 @@ def extract_number(text, short_scale=True, ordinals=False, lang=None):
     return text
 
 
-def extract_duration(text, lang=None):
+def extract_calendar_duration(text, lang=None, replace_token=""):
```

Comment:
Would recommend calling this `extract_offset`, or `extract_relativedelta`.

Reply:
I'm all for something less clunky than `extract_calendar_duration`, but I'd like it if all the functions in LF had such painfully descriptive names, somebody can figure them out who has never written code before.